src/include/miopen/solver.hpp

```diff
@@ -853,24 +853,14 @@ struct ConvHipImplicitGemmV4R4WrW : SolverBase<ConvolutionContext>
     ConvSolution GetSolution(const ConvolutionContext& ctx,
                              const PerformanceImplicitGemmV4R4WrW& config,
                              bool disableConfigOverrideFromEnv = false) const;
-
-    protected:
-    bool IsApplicableMlirCommon(const ConvolutionContext& ctx) const;
 };
 
-class ConvHipImplicitGemmMlirCppWrW : ConvHipImplicitGemmV4R4WrW
+// Mlir based igemm kernels do not support tuning, for now
```

Comment:
Please remove the comment -- it is trivial ;)

Reply:
Will get rid of them